### ISSUE TITLE
OpenBSD is not supported, yet glow and nvim is?

### ISSUE THREAD

**Issue by anoduck** [ellisonleao/glow.nvim#92]
Installed package, hoping to use it to render markdown files. It should have been a sure thing, since both neovim, glow, and lua are supported in OpenBSD. Yet, when I went to generate the preview, I received the error, `os not supported`.  So, I am looking for a reason as to why I received this message, and obviously why OpenBSD is not supported?

**Comment by ellisonleao** [comment]
hey @anoduck do you already have the glow binary installed?

**Comment by anoduck** [comment]
> hey @anoduck do you already have the glow binary installed?

Yes, I do in `~/go/bin/bin/glow`.

**Comment by ellisonleao** [comment]
@anoduck @Ghasak that config no longer exists. If you have `glow` binary in your `$PATH`, the plugin should automatically find it for you. If not, it will try to install in the `install_path` (Just updated README doc) config var

**Comment by anoduck** [comment]
@ellisonleao sadly, this is not working. I am still receiving the `os unsupported` error. Are there any special configuration requirements to setup go to work with neovim?


Here is my configuration for packer:

```lua
-- use glow for markdown preview
        use {
	     'ellisonleao/glow.nvim',
	         config = function()
			require("glow").setup {
			   glow_path = "/home/user/go/bin/bin/glow",
			   install_path = "/home/user/.local/bin",
			   style = 'dark',
			   pager = false,
			   width = 80,
			   }
		 end
	  }
```

**Comment by ellisonleao** [comment]
the `os not supported` msg happens when the plugin does not identify the glow binary installation and tries to install it. The install script tries to install it, but it can't identify your OS, then it outputs that msg. but there's seems to be a bug in the `glow_path` check as well which I will investigate. Also you don't need the default field values in your setup call, it should be something like:

```lua
        use {
	     'ellisonleao/glow.nvim',
	         config = function()
			require("glow").setup {
			   glow_path = vim.env.HOME.. "/go/bin/bin/glow",
                           style = 'dark',
                           width = 80
			 }
		 end
	  }
```

 In the meantime can you confirm if `glow` binary is in your `$PATH`? Can you show the output for the command:

```
:lua print(vim.fn.exepath("glow"))
```
in your machine?

**Comment by anoduck** [comment]
@ellisonleao Eureka! AND Ah, Crap! Foiled again!

I found it, I found it, ah shit, I really found it. 

Why does it always have to be user error? More importantly, why does it have to be my error?

The error is the result of a missing `:` while defining `vim.env.PATH`. This missing character prevented neovim's path from properly being configured, so neovim never was able to pick up on any of my go binaries. This explains more than just the experienced issue discussed here. 

Wow, really swanky preview there. I love the popup window. Super impressed! Great work.

I am off to see what else my user error was preventing from running. Cheers!
